ubg/.gitignore

```diff
@@ -23,3 +23,4 @@ yarn-debug.log*
 yarn-error.log*
 
 .env
+apiGoogleconfig.json
```

Comment:
Maybe you could use the same .env file we have for our firebase credentials instead of creating a new file.